src/main/resources/openapi/cases.yaml

```diff
@@ -270,36 +270,6 @@ paths:
               schema:
                 $ref: './problem.yaml'
 
-
-  /cases/hearings/{hearing_id}/events:
-    get:
-      tags:
-        - Cases
-      summary: Allows the retrieval of events for a given hearingId
-      operationId: getEvents
-      description: |-
-        Retrieves all events for a given hearingId.
-      parameters:
-        - in: path
-          name: hearing_id
-          schema:
-            type: integer
-          description: "hearingId is the internal hea_id of the hearing."
-          required: true
-      responses:
-        '200':
-          description: OK
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/events_response'
```

Comment:
If the `events_response` is no longer used it can be nuked also (lines 523-543).